website_user_login_redirect_custom/controllers/main.py

```
@@ -0,0 +1,67 @@
+        return bool(str2bool(enabled))
+
+    def _get_custom_redirect_url(self):
+        """Get custom redirect URL"""
+        return (
+            request.env["ir.config_parameter"]
+            .sudo()
+            .get_param("website_user_login_redirect_custom.url", "/")
+        )
+
+    @http.route("/web/login", type="http", auth="public", website=True, sitemap=False)
+    def web_login(self, redirect=None, **kw):
```

Ruff (0.14.0)

Missing type annotation for `**kw`

(ANN003)